### PR TITLE
Dry up refresh function

### PR DESCRIPTION
**Pull request by dleard**

### CODE REVIEW

**Comment by dleard** [comment]
@wenzowski 
I've DRY'd up the function as we discussed. A couple thoughts: 

1) I packaged it back inside the export function now that we removed `flat`. I could move the array of matviews inside the refresh function to make it standalone again if we like that better?

2) I think that running the refresh function in a loop maybe blocks us from doing the refreshes in parallel?
https://www.postgresql.org/docs/9.6/when-can-parallel-query-be-used.html

File: deploy/function_export_mv_to_table.sql [hunk 1]
```diff
@@ -21,12 +21,17 @@ $$
                        'final_report, fuel, permit, parent_organisation, contact, ' ||
                        'address, descriptor, measured_emission_factor}';
 
-  -- Function refresh_materialized_views(data boolean) must be run before this function
   begin
 
+    -- Refresh materialized views
     for i in 1 .. array_upper(mv_array, 1)
       loop
+        perform ggircs_swrs.refresh_materialized_views(quote_ident(mv_array[i]), 'with data');
```
Comment (wenzowski) [inline deploy/function_export_mv_to_table.sql]: so this has the side-effect of removing the refresh on `flat` :)

File: test/unit/function_export_mv_to_table_test.sql [hunk 2]
```diff
@@ -744,17 +744,23 @@ $$), ($$
...
+                       activity, unit, identifier, naics, emission,
+                       final_report, fuel, permit, parent_organisation, contact,
+                       address, descriptor, measured_emission_factor}$$;
+    begin
+      for i in 1 .. array_upper(mv_array, 1)
+          loop
+            perform ggircs_swrs.refresh_materialized_views(quote_ident(mv_array[i]), 'with data');
+          end loop;
+    end
+$do$;
```
Comment (wenzowski) [inline test/unit/function_export_mv_to_table_test.sql]: makes sense

File: test/unit/function_export_mv_to_table_test.sql [hunk 3]
```diff
@@ -1856,7 +1860,22 @@ select results_eq(
...
+      declare mv_array text[] := $$
+                       {report, organisation, facility,
+                       activity, unit, identifier, naics, emission,
+                       final_report, fuel, permit, parent_organisation, contact,
+                       address, descriptor, measured_emission_factor}
+                       $$;
+      begin
+        for i in 1 .. array_upper(mv_array, 1)
+          loop
+            perform ggircs_swrs.refresh_materialized_views(quote_ident(mv_array[i]), 'with no data');
```
Comment (wenzowski) [inline test/unit/function_export_mv_to_table_test.sql]: test that data is cleared 👍

**Comment by wenzowski** [comment]
rebased

**Pull request merged**